.github/workflows/test.yml

```diff
@@ -78,3 +78,14 @@ jobs:
       run: |
         ./verify.sh cluster
 
+  # test if kernel module can be loaded, this is very os and instance specific, ignore if it fails
+  test_kernel_module:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: ./
+      with:
+        kernel_module_names: rapl,intel_rapl_common
+    - name: verify
+      run: |
+        ./verify.sh modprobe rapl,intel_rapl_common
```

Comment:
It seems that the logic here is always run:
 ```
apt-get install -y linux-modules-`uname -r` linux-modules-extra-`uname -r`
```
in GHA, and the `verify.sh` just throws some err msg in console if any of the module in `kepler_module_names` is N/A.

Can we just merge the `verify.sh` logic into the `index.js` and only install modules when RAPL related modules are N/A?
And if the "always install linux modules" logic is acceptable, can we just ignore the "verify.sh modprobe" logic then?

Reply:
moved to `env`